data/assets/scene/solarsystem/heliosphere/bastille_day/fluxnodescutplane.asset

```diff
@@ -0,0 +1,113 @@
+local assetHelper = asset.require("util/asset_helper")
+local propertyHelper = asset.require('../../../../util/property_helper')
+local transforms = asset.require('scene/solarsystem/sun/transforms_heliosphere')
+
+local TexturesPathEquitorial = asset.syncedResource({
+Type = "HttpSynchronization",
+  Name = "cutplanes_textures",
+  Identifier = "cutplanes_textures",
+  Version = 1
+})
+
+local TexturesPathMeridial = asset.syncedResource({
+  Type = "HttpSynchronization",
+  Name = "cutplane_meridial_textures",
+  Identifier = "cutplane_meridial_textures",
+  Version = 1
+})
+
+local EquatorialCutplane = {
+  Identifier = "EquatorialCutplane",
+  Parent = transforms.HEEQ180ReferenceFrame.Identifier,
+  Transform = {
+    Rotation = {
+    -- This is a rotation matrix to go from Carrington longitude referens frame to HEEQ180
+    -- which is what the renderables parent transforms.HEEQ180ReferenceFrame. Identifier is
+    -- referring to. At the reference time, MAS_seq = 0, 2000-07-14T08:33:37.105 the Carrington
+    -- longitude was 309.3 degrees.
+    -- Difference from HEEQ => 360-309.3=50.7
+    -- (or 0-309.3 = -309.3 However this leads to the same rotation matrix in the end)
+    -- Since OpenSpace supports HEEQ180 and not HEEQ, 180 was added or subtracted
+    -- => a1 = -129.3 and a2 = 230.7
+    -- Rotation matrix: (cos a, -sin a, 0)(sin a, cos a, 0)(0, 0, 1) leads to the result.
```

Comment:
Indent comments one more step

Reply:
Also still wondering if this table for the rotation (which is used in three assets) could be put into its own asset file 🤔 like a 
```
local CarringtonLongitudeToHEEQ180Rotation = { 
  ... --the table you have here, with comments and all
}
 
asset.export("CarringtonLongitudeToHEEQ180Rotation", CarringtonLongitudeToHEEQ180Rotation)
```

Could be nice if someone else wants to use the same rotation for something. But I guess we donät really do this anywhere else so don't know if it really makes sense